node/src/db.rs

```diff
@@ -134,6 +159,17 @@ impl SecretDBUpdate {
         self.batch.delete_cf(&self.db.cf_handle(col), key);
     }
 
+    pub fn delete_all(&mut self, col: DBCol) -> anyhow::Result<()> {
+        let range = self.db.get_cf_key_range(col)?;
+        if let Some(range) = range {
+            self.batch
+                .delete_range_cf(self.db.cf_handle(col), range.start(), range.end());
+            // delete_range_cf deletes ["begin_key", "end_key"), so need one more delete
```

Comment:
It's surprising there isn't a nicer way of doing this. One recommendation I saw is to pass (smallest possible key, largest possible key) but that is not that great either.

Reply:
Same here, copied from nearcore. I'm surprised too....

Reply:
The problem is the smallest key is b"", but there does not exist a largest possible key.